authentik/stages/identification/stage.py

```diff
@@ -237,7 +237,7 @@ def get_challenge(self) -> Challenge:
         )
         for source in sources:
             ui_login_button = source.ui_login_button(self.request)
-            if source.component == "":
+            if source.slug != "authentik-built-in" and source.component == "":
                 continue
```

Comment:
```suggestion
```

This entire block can actually be removed as the check isn't required, due to `.ui_login_button` returning None